app/policies/application_policy.rb

```diff
@@ -43,7 +43,7 @@ def initialize(user, scope)
     end
 
     def resolve
-      scope.all
+      scope.all if user.is_a?(AdminUser)
```

Comment:
What happens if the user is not an admin? This is used across more places and I think the default is alright, it'd just be overridden in other policies if you wish to limit what the user can see.

We are also violating LSP, we should at least return `scope.none` to not break that principle. But I think it still makes more sense to return all records.